tst/testinstall/cache.tst

```diff
@@ -0,0 +1,29 @@
+#############################################################################
+##
+#W  cache.tst                GAP Library                      Chris Jefferson
```

Comment:
Ho-hum...
1. the commit message says "Add test for CYCLOTOMIC_FIELDS";
2. but the commits adds a file `cache.tst`;
3. but that actually tests `CyclotomicField`

Something is not right here :-). It fails to do 1, because `CYCLOTOMIC_FIELDS` is never tested explicitly, so this relies on implementation details which might change. Similar for the 2nd point: this is not a test for the cache code, as this is an implementation detail of `CyclotomicField` which might change. 

So perhaps this test file should be renamed to `tst/testinstall/opers/CyclotomicField.tst`; then a new `cache.tst` file could be added, which actually tests `MemoizePosIntFunction` explicitly, by calling it.

Reply:
Moved and renamed test. MemoizePosIntFunction is tested by a manual example.